tap_zendesk/schemas/call_legs.json

```diff
@@ -2,105 +2,186 @@
     "type": "object",
     "properties": {
         "agent_id": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],
```

Comment:
Can we only include null for the fields that might return null? The docs don't say which fields return null or not, but some of these must be filled in (eg. the `id` or `leg_id` field) https://developer.zendesk.com/api-reference/voice/talk-api/incremental_exports/#example-responses-1

Reply:
i am going to keep them all as null possible except PK aka id and leg_id